mod/members/languages/en.php

```diff
@@ -9,6 +9,7 @@
 	'members:label:online' => 'Online',
 	'members:label:search' => 'Search results',
 	'members:search' => 'Search members',
+	'members:aria:label:user_search' => "Search for users",
```

Comment:
should it not be search for members? than also update the label key

Reply:
fixed